src/mym.cpp

```diff
@@ -1353,10 +1353,8 @@ char* serializeString(size_t &rnBytes, const mxArray*rpArray, const char*rpArg,
     const mwSize n_dims = mxGetNumberOfDimensions(rpArray);
     char* p_buf = NULL;
     if (mxIsEmpty(rpArray)) {
-        p_buf = (char*)mxCalloc(2, sizeof(char));
-        p_buf[0] = ' ';
-        p_buf[1] = 0;
-        rnBytes = 1;
+        p_buf = (char*) '\0';

```

Comment:
wow so that was there from the very beginning.  I wonder why. Thanks for the fix.